backend/src/main/java/heartbeat/service/board/jira/JiraService.java

```diff
@@ -530,10 +529,22 @@ private List<String> getAssigneeSet(StoryPointsAndCycleTimeRequest request, URI
 	}
 
 	private List<String> getHistoricalAssignees(URI baseUrl, String cardKey, String token) {
-		CardHistoryResponseDTO jiraCardHistory = jiraFeignClient.getJiraCardHistory(baseUrl, cardKey, token);
+		CardHistoryResponseDTO jiraCardHistory = getCardAllHistory(baseUrl, cardKey, 0, token);
 		return jiraCardHistory.getItems().stream().map(item -> item.getActor().getDisplayName()).distinct().toList();
 	}
 
+	private CardHistoryResponseDTO getCardAllHistory(URI baseUrl, String cardKey, int startAt, String token) {
+		int pageCount = 100;
```

Comment:
how to handle the page of `200`.....

Reply:
Get all data recursively, 100 items each time